tests/framework/installer/installer.go

```diff
@@ -46,13 +47,34 @@ var (
 	createFromStdinArgs = append(createArgs, "-")
 	deleteArgs          = []string{"delete", "-f"}
 	deleteFromStdinArgs = append(deleteArgs, "-")
+	// test suite names
+	CassandraTestSuite   = "cassandra"
```

Comment:
Shouldn't these be constants?

Reply:
yep, copy/pasted to the wrong section...